### PR TITLE
Secret bootstrapper: Allow finding config entry for faulty dockercfg

### PR DESCRIPTION
**Pull request by alvaroaleman**
The current error is pretty much useless for finding the offending item

/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

File: cmd/ci-secret-bootstrap/main.go [hunk 1]
```diff
@@ -326,7 +326,8 @@ func constructSecrets(ctx context.Context, config secretbootstrap.Config, bwClie
 	errChan := make(chan error, potentialErrors)
 
 	secretConfigWG := &sync.WaitGroup{}
-	for _, cfg := range config.Secrets {
+	for idx, cfg := range config.Secrets {
+		idx := idx
```
Comment (droslean) [inline cmd/ci-secret-bootstrap/main.go:330]: is this needed?

Comment (alvaroaleman) [reply cmd/ci-secret-bootstrap/main.go:330]: Yes

**Review by droslean — APPROVED** [review]

**Pull request merged**